include/albatross/models/sparse_gp.h

```diff
@@ -202,14 +224,15 @@ class SparseGaussianProcessRegression
      *
      *     v = L^-T B^-1 P * A^-1 y
      */
-    Eigen::VectorXd A_sqrt = A.array().sqrt();
-    Eigen::MatrixXd RtR = A_sqrt.asDiagonal().inverse() * P.transpose();
+    auto A_llt = A.llt();
```

Comment:
Could this be less efficient if it's diagonal? I assume `Eigen` will make sure it isn't...

Reply:
Yeah, it might be slightly less efficient ... but I didn't notice any significant difference in the unit tests.